server/routerlicious/packages/services-client/src/error.ts

```diff
@@ -86,7 +100,23 @@ export class NetworkError extends Error {
             message: this.message,
             canRetry: this.canRetry,
             isFatal: this.isFatal,
-            retryAfter: this.retryAfterMs,
+            retryAfter: this.retryAfterMs !== undefined ? this.retryAfterMs / 1000 : undefined,
```

Comment:
Can you just do `retryAfter: this.retryAfter`, since you did this computation in the constructor to set `this.retryAfter`?

Reply:
Yes! Thank you, good catch